### PR TITLE
Improving code style : verbosity, indentation levels

### PR DESCRIPTION
**Pull request by garsaud**
Fixing some nested if() statements and reducing indentation levels.

Making some sections less verbose, e.g:
```cpp
if (a == 1)
{
    b = true;
}
else
{
    b = false;
}
```
↓
```cpp
b = (a == 1)
```

Changes are best viewed ignoring whitespaces:

### CODE REVIEW

File: src/CalcManager/UnitConverter.cpp [hunk 1]
```diff
@@ -505,142 +498,144 @@ void UnitConverter::SendCommand(Command command)
 {
     if (CheckLoad())
     {
-        //TODO: Localization of characters
-        bool clearFront = false;
-        if (m_currentDisplay == L"0")
+        return;
```
Comment (HowardWolosky) [inline src/CalcManager/UnitConverter.cpp]: This change is incorrect.  You didn't invert the conditional check above.

Comment (HowardWolosky) [inline src/CalcManager/UnitConverter.cpp]: You missed inverting the conditional above.

Comment (garsaud) [reply src/CalcManager/UnitConverter.cpp]: Yes indeed.
Fixed

Comment (garsaud) [reply src/CalcManager/UnitConverter.cpp]: Thanks. I inverted it in commit cda181e

**Comment by HowardWolosky** [comment]
@garsaud -- Can you please sync and merge your branch with the latest changes from master.  There are currently conflicts.

**Comment by garsaud** [comment]
@HowardWolosky — Conflicts have been fixed

**Comment by HowardWolosky** [comment]
@garsaud - It looks like conflicts have come in since this was approved. Can you please resolve the conflicts, and then we can get this checked-in.  Thanks!

**Comment by garsaud** [comment]
@HowardWolosky – Conflicts are resolved. Thanks!

**Comment by HowardWolosky** [comment]
Closing and re-opening in order to re-kick off the CLA validation check.

**Pull request merged**